src/middlewared/middlewared/alert/source/truenas_verify.py

```diff
@@ -0,0 +1,32 @@
+from datetime import timedelta
+import subprocess
+
+from middlewared.alert.base import AlertClass, AlertCategory, Alert, AlertLevel, ThreadedAlertSource
+from middlewared.alert.schedule import IntervalSchedule
+
+
+# --------------- Monitored Alerts ----------------
+class TrueNASVerifyServiceChangeDetectionAlertClass(AlertClass):
+    category = AlertCategory.AUDIT
+    level = AlertLevel.ERROR
+    title = "TrueNAS Verify Service: Changes detected in root file system."
+    text = "%(verrs)s"
+
+
+class TrueNASVerifyServiceChangeDetectionAlertSource(ThreadedAlertSource):
+    '''
+    Periodic verification of root file system
+    '''
+    schedule = IntervalSchedule(timedelta(hours=24))
+    run_on_backup_node = False
+
+    def check_sync(self):
```

Comment:
Per discussion earlier we should check whether STIG is enabled before running this. We'll use the ixdiagnose results containing this as a litmus test for fine-tuning the verify results.